### DISCUSSION TITLE
passing -Dxxx to the cmake config of a package

### DISCUSSION THREAD

**Discussion by grbba** [cpm-cmake/CPM.cmake#232] [Q&A] [answered]
I guess it must be obvious but i can't really find my way around .. How do i pass -Dxxxx or -Dxxxx=yyy to the package loaded by CPMAddPackage for configuring the cmake scripts of the package loaded ?

Thanks a lot

**Comment by JohelEGP** [comment, marked as answer]
I think what you're looking for is `OPTIONS`, see the README.
https://github.com/cpm-cmake/CPM.cmake/blob/8e8dcc9a8dbd403118da7a867d3f1c0d43ed922f/README.md#L34-L42
Check out the README and wiki for examples.

**Comment by grbba** [comment]
Thanks i was thinking that for OPTIONS you always have to have a key value pair for any option so therefore my question. I tried now to just pass the key and that seems to work 💯 
Thanks a lot
